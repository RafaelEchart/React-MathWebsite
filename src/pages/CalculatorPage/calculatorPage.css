.calculatorContent_Container {
  margin: 5%;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
}

.calculatorTitle {
  margin: 20px;
}
