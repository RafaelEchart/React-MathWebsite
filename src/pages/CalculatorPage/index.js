import React from 'react';
import Calculator from '../../components/Calculator';
import './calculatorPage.css';

const CalculatorPage = () => (
        <div className="calculatorContent_Container" >
          <h2 className="calculatorTitle">Lets do some math!</h2>
          <Calculator/>
        </div>
);
export default CalculatorPage;