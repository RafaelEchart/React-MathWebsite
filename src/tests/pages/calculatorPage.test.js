import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorPage from '../../pages/CalculatorPage/index.js';

describe('Snapshot test for Calculator Page', () => {
  it('Should render CalculatorPage', async () => {
    const tree = renderer.create(<CalculatorPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
