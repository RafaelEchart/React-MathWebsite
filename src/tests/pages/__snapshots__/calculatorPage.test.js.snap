// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Calculator Page Should render CalculatorPage 1`] = `
<div
  className="calculatorContent_Container"
>
  <h2
    className="calculatorTitle"
  >
    Lets do some math!
  </h2>
  <div
    className="calculator_container"
  >
    <div
      className="result_container"
    >
      0
    </div>
    <div
      className="row_container"
    >
      <button
        className="number gray_button"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="operation"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="operation"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="operation"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
    </div>
    <div
      className="row_container"
      type="button"
    >
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="operation"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
    </div>
    <div
      className="row_container"
    >
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="operation"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
    </div>
    <div
      className="row_container"
    >
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="operation"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      className="row_container"
    >
      <button
        className="number_zero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="number"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="operation"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
